partner_multi_relation/views/res_partner.xml

```diff
@@ -1,43 +1,53 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These are no longer needed since v11 I believe but don't impact functionality leaving them there.

Reply:
> These are no longer needed since v11 I believe but don't impact functionality leaving them there.

Don't misunderstand python files encoding and xml document encoding which is not equal!